deployments/values/supported-matrix.yaml

```diff
@@ -4,7 +4,7 @@
 patterns:
   ga: &ga_pattern (\.[0-9]+){1,2}
   ga_with_hotfixes: &ga_hotfixes_pattern (\.[0-9]+){0,3}
-  development: &development_pattern (\.[0-9]+){0,3}(-[AM]\.?[0-9]+)?
+  development: &development_pattern (\.[0-9]+){0,3}(-A\.?[0-9]+)?
```

Comment:
it wasn't selecting the last based on date? (whatever A or M was)

Reply:
no it's using alphabetical sorting: M1 > A9 > A14 👎